src/Models/Site.php

```diff
@@ -348,6 +351,7 @@ public function serialize()
             'holder_id' => $this->get('holder_id'),
             'owner' => $this->get('owner'),
             'frozen' => $this->isFrozen() ? 'true' : 'false',
+            'last_frozen_at' => $format_date('last_frozen_at'),
```

Comment:
Using a dynamic function and a closure here feels a bit javascript-y but it works. It's a whole new PHP world. I'm ok with this.